src/user/user.spec.js

```diff
@@ -54,4 +54,24 @@ describe("user :", () => {
       await expect(xo.createUser(simpleUser)).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".delete() :", () => {
+    it("deletes a user successfully with id", async () => {
+      const userId = await xo.call("user.create", simpleUser);
+      expect(await xo.call("user.delete", { id: userId })).toBeTrue();
+      expect(find(await xo.getAllUsers(), { id: userId })).toBeUndefined();
```

Comment:
I think you can also remove `getAllUsers` and use `call('user.getAll')` directly.